Util/UserManipulator.php

```diff
@@ -55,68 +97,106 @@ public function create($username, $password, $email, $active, $superadmin)
         $user->setSuperAdmin((Boolean) $superadmin);
         $this->userManager->updateUser($user);
 
+        $event = new UserEvent($user, $this->getRequest());
+        $this->dispatcher->dispatch(FOSUserEvents::USER_CREATED, $event);
+
         return $user;
     }
 
     /**
      * Activates the given user.
      *
      * @param string $username
+     *
+     * @return \FOS\UserBundle\Model\UserInterface
      */
     public function activate($username)
     {
         $user = $this->findUserByUsernameOrThrowException($username);
         $user->setEnabled(true);
         $this->userManager->updateUser($user);
+
+        $event = new UserEvent($user, $this->getRequest());
+        $this->dispatcher->dispatch(FOSUserEvents::USER_ACTIVATED, $event);
+
+        return $user;
```

Comment:
Yes, I use to write fluent methods, but maybe it has no sense here, I've removed the returns.